### ISSUE TITLE
not finding issues which are available.

### ISSUE THREAD

**Issue by stustaff** [evilhero/mylar#973]
Im not sure whats happening, couple of examples.

Wolverines 13 is available but mylar doesnt find it.
All almost 50 issues of futures end ARE available it has found 1. heres an example
015-04-04 12:13:18  INFO    Completed RSS Search scan
2015-04-04 12:13:18     INFO    Finished searching via :rss. Issue not found - status kept as Wanted.
2015-04-04 12:13:18     INFO    Could not find Issue 48 of The New 52: Futures End(2014) using geek2
2015-04-04 12:13:18     INFO    Shhh be very quiet...I'm looking for The New 52 Futures End issue: 48 (2015) using geek2(newznab) [RSS]
2015-04-04 12:13:18     INFO    Alternate Search pattern detected...re-adjusting to : The New 52 Futures End 2015
2015-04-04 12:13:18     INFO    Shhh be very quiet...I'm looking for The New 52: Futures End issue: 48 (2015) using geek2(newznab) [RSS]

the file is on nzbgeek under both:-
The New 52 - Futures End 048 (2015) (digital-Empire)
and
The.New.52.-.Futures.End.048.2015.digital-Empire

the one it has found is named
The New 52 - Futures End 044 (2015) (digital-Empire)

which i find odd.?

its happening across multiple series I get some but not others eg 1 out of the three new deadpool series

i cant identify why its not getting them.

ta

**Comment by evilhero** [comment]
I'm not saying it's the problem, but it looks like you're only doing rss searches (like you're clicking on the rss check option in the configuration under the providers tab), or you're just seeing the log results from an rss search which happens every 20 minutes and those series haven't been recently posted to Usenet. 

Have you tried doing a Force Search option from the Wanted tab? Or trying an individual issue by clicking on the magnifying glass icon (which does a full search, but doesn't mark it as Wanted), or the checkmark icon (marks it as Wanted), or the checkbox(es)  to mark issues as Wanted. All of these options run a full search - meaning it checks the rss first for the release, and then it drops down to api search for deeper searching. Full search  also runs every 6 hrs by default so as to not hammer the indexers and kill your api hits ;)

**Comment by stustaff** [comment]
i had reopened and pressedforce check from wanted tab when i went to log and got that text.
I just used magnifying glass in wanted tab and got it.

why did that work but not fetching them automatically?
is it because i havent left it long enough after all editing etc of config and restarting?

**Comment by stustaff** [comment]
ok so force check still didnt find anything but log is different.

2015-04-04 15:13:21 	INFO 	Finished searching via :rss. Issue not found - status kept as Wanted.
2015-04-04 15:13:21 	INFO 	Could not find Issue 42 of The New 52: Futures End(2014) using geek2
2015-04-04 15:13:21 	INFO 	Shhh be very quiet...I'm looking for The New 52 Futures End issue: 42 (2015) using geek2(newznab) [RSS]
2015-04-04 15:13:21 	INFO 	Alternate Search pattern detected...re-adjusting to : The New 52 Futures End 2015
2015-04-04 15:13:21 	INFO 	Shhh be very quiet...I'm looking for The New 52: Futures End issue: 42 (2015) using geek2(newznab) [RSS]
2015-04-04 15:13:21 	INFO 	search provider order is ['newznab:geek2']
2015-04-04 15:13:21 	INFO 	Issue Title given as : None
2015-04-04 15:13:21 	INFO 	[SEARCH] Special Characters exist within Series Title. Enabling search-safe Name : The New 52 Futures End

**Comment by evilhero** [comment]
Both the magnifying glass & the Wanted tab use the same function of calls, so there should be no difference - you'll have to post the mylar.log file contents showing the searching. The GUI log doesn't help very much when it comes to debugging things like this ;)

**Comment by stustaff** [comment]
ok i wll leave it running for a while.
it seems to be finding the odd one but sending to sabnzb twice creating duplicates.
do i post the whole log somewhere or just a relevant section in here?

**Comment by evilhero** [comment]
Just the relevant portion is all that's needed I think, from the start of the search to the point where it says it's finished. 

Which branch /commit are you on of mylar?

**Comment by stustaff** [comment]
thank for the help

Mylar Version: development
-- source build 87d37e6011ca79b78160264e652bcff553350c26.
Python Version : 2.7.1

Ok so issue IS available "Injustice - Gods Among Us - Year Two 009 (2014) (digital) (Son of Ultron-Empire)"

log file shows - 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : checking search result: DC Digital-Firsts [3 of 14] Injustice - Gods Among Us - Year Two 009 (2014) (digital) (Son of Ultron-Empire)
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : size given as: 24.5 MB
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Tue, 20 May 2014 11:32:55 +0000 is after store date of 2014-02-12
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Entry: DC Digital-Firsts [3 of 14] Injustice - Gods Among Us - Year Two 009 (2014) (digital) (Son of Ultron-Empire)
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Cleantitle: dc digital-firsts 3 of 14 injustice - gods among us - year two 009 (2014) (digital) (son of ultron-empire)
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Cleantitle: dc digital-firsts 3 of 14 injustice - gods among us - year two 009 (2014) (digital) (son of ultron-empire)
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 0. Bracket Word: dc digital-firsts 3 of 14 injustice - gods among us - year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Comic: dc digital-firsts 3 of 14 injustice - gods among us - year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : UseFuzzy is  : None
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : ComVersChk : 0
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Series version detected as V1 (only series in existance with that title). Bypassing Year/Volume check
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : mini-series detected : dc digital-firsts 3 of 14 injustice - gods among us - year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Scanner detected: dc digital-firsts 3 of 14 injustice - gods among us - year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 1. Bracket Word: 2014
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : year detected: 2014
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : year looking for: 2014
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 2014 - right years match baby!
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 3. Bracket Word: digital
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : digital edition detected
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 5. Bracket Word: son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : mini-series detected : son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Scanner detected: son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : original nzb comic and issue: dc digital-firsts 3 of 14 injustice - gods among us - year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : [Injustice: Year Two] I have found a -  within the nzbname @ position: 10
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : There is no hyphen present in the series title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Assuming position start is : 10
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : First word of issue stored as : 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : adjusted nzb comic and issue: dc digital firsts 3 of 14 injustice   gods among us   year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : adjusting from: dc digital firsts 3 of 14 injustice   gods among us   year two 009  to: 009
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : ['dc', 'digital', 'firsts', '3', 'of', '14', 'injustice', 'gods', 'among', 'us', 'year', 'two', '009'] nzb series word count: 12
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : ['injustice', 'year', 'two'] watchlist word count: 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : incorrect comic lengths...not a match
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : there are 2 words in the issue title of : protection; duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : splitit:['dc', 'digital', 'firsts', '3', 'of', '14', 'injustice', 'gods', 'among', 'us', 'year', 'two', '009']
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : splitst:12
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : len-watchcomic:3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : possible issue number of : 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : there are 9 left over after we remove the series title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : search_issue_title is : 3 of 14 injustice gods among us year two 009
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : possible issue number of : 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : hypen split detected.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : located first word of : 3 at position : 0
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : corrected search_issue_title is now : 3 of 14 injustice gods among us year two 009
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 3 -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : of -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 14 -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : injustice -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : gods -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : among -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : us -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : year -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : two -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 009 -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 3 -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : of -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 14 -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : injustice -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : gods -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : among -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : us -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : year -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : two -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 009 -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : isstitle_match count : 0
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 0 words matched on issue title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : No issue title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : splitst : 12
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : len-watchcomic : 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : incorrect comic lengths after removal...not a match.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : checking search result: Injustice-Gods.Among.Us-Year.Two.009.2014.digital.Son.of.Ultron-Empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : size given as: 24.5 MB
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Tue, 20 May 2014 11:32:57 +0000 is after store date of 2014-02-12
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Entry: Injustice-Gods.Among.Us-Year.Two.009.2014.digital.Son.of.Ultron-Empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Cleantitle: injustice-gods among us-year two 009 2014 digital son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : detected invalid nzb filename - attempting to detect year to continue
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Cleantitle: injustice-gods among us-year two 009 (2014)  digital son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 0. Bracket Word: injustice-gods among us-year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Comic: injustice-gods among us-year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : UseFuzzy is  : None
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : ComVersChk : 0
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Series version detected as V1 (only series in existance with that title). Bypassing Year/Volume check
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 1. Bracket Word: 2014
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : year detected: 2014
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : year looking for: 2014
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 2014 - right years match baby!
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 2. Bracket Word:   digital son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : mini-series detected :   digital son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Scanner detected:   digital son of ultron-empire
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : original nzb comic and issue: injustice-gods among us-year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : [Injustice: Year Two] I have found a -  within the nzbname @ position: 9
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : There is no hyphen present in the series title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : Assuming position start is : 9
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : First word of issue stored as : among
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : adjusted nzb comic and issue: injustice gods among us year two 009 
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : adjusting from: injustice gods among us year two 009  to: 009
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : ['injustice', 'gods', 'among', 'us', 'year', 'two', '009'] nzb series word count: 6
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : ['injustice', 'year', 'two'] watchlist word count: 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : incorrect comic lengths...not a match
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : there are 2 words in the issue title of : protection; duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : splitit:['injustice', 'gods', 'among', 'us', 'year', 'two', '009']
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : splitst:6
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : len-watchcomic:3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : possible issue number of : us
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : there are 3 left over after we remove the series title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : search_issue_title is : us year two 009
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : possible issue number of : us
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : hypen split detected.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : located first word of : among at position : -1
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : corrected search_issue_title is now : 9
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 9 -TO- protection;
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : looking at : 9 -TO- duty
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : isstitle_match count : 0
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : 0 words matched on issue title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : No issue title.
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : splitst : 6
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : len-watchcomic : 3
04-Apr-2015 16:02:04 - DEBUG   :: Thread-12 : incorrect comic lengths after removal...not a match.

**Comment by evilhero** [comment]
Injustice is one of those series that's really messed up, between the digital and print versions and how they are posted Vs how comicvine has them listed as. 

Which provider are you using? The reason I ask is if it's not the experimental option then they need to update their regexs to parse out the case at the beginning.

**Comment by stustaff** [comment]
yeh i realised that might be a bad example as the names are bonkers
it is 'everything' though
eg wolverines 11 which is pretty simple.

let me get a log file and info for one of the straightforward ones.
provider is extreme usenet.

**Comment by evilhero** [comment]
Sorry I didn't mean your usenet provider, I meant your nzb provider that mylar is pulling the search results from ;)

**Comment by stustaff** [comment]
Sorry nzbgeek they are pretty good and very active.

**Comment by evilhero** [comment]
I'm going to guess that most of your search results have some crap before the actual filename, which is standard for raw indexers or Newznabs that don't apply the correct filters to the headers they download. 

I mean, they clean up movie titles so it's just the title, but comics they don't and leave all the crap in - when was the last time you browsed the movie section of your nzb provider and saw 'Digital HD RELEASE by DaCreW' or something at the start of the name? (sorry, not your fault just a problem with a great majority of indexers). 

If you can post what mylar is trying to parse the search results for, I can try to apply filters so they catch them abit better possibly.

**Comment by stustaff** [comment]
not sure? they are all comic - issue title - number, nothing messy and only ever release name etc in brackets at end.

eg batman - arkham knight - 006 (superreleaseteam)
or a variation with full stops instead spaces etc.

uncanny xmen 24 has three versions on nzb geek file name shows as:- 
0-Day 2014.7.30 - "Uncanny X-Men 024 (2014) (Digital) (Zone-Empire).cbr" yEnc (1/44)

this gave this and didnt grab it the file size listed in log is correct for the file so it did find it but then stopped?? 

Comic: uncanny x-men 024 

is there some feedback I can give them on making files clearer for grabbing?

04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : UseFuzzy is  : None
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : ComVersChk : 3
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 1. Bracket Word: 2014
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : year detected: 2014
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : year looking for: 2014
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 2014 - right years match baby!
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 3. Bracket Word: digital
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : digital edition detected
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 5. Bracket Word: zone-empire
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Scanner detected: zone-empire
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : original nzb comic and issue: uncanny x-men 024 
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : [X-Men] I have found a -  within the nzbname @ position: 9
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : There is a - present in the series title. Ignoring position: 9
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : adjusted nzb comic and issue: uncanny x men 024 
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : adjusting from: uncanny x men 024  to: 024
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : ['uncanny', 'x', 'men', '024'] nzb series word count: 3
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : ['x', 'men'] watchlist word count: 2
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : splitst : 3
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : len-watchcomic : 2
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : incorrect comic lengths after removal...not a match.
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : checking search result: Uncanny.X-Men.024.2014.Digital.Zone-EMPIRE
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : size given as: 27.4 MB
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Thu, 31 Jul 2014 02:29:03 +0000 is after store date of 2014-05-21
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Entry: Uncanny.X-Men.024.2014.Digital.Zone-EMPIRE
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Cleantitle: uncanny x-men 024 2014 digital zone-empire
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : detected invalid nzb filename - attempting to detect year to continue
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Cleantitle: uncanny x-men 024 (2014)  digital zone-empire
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 0. Bracket Word: uncanny x-men 024 
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Comic: uncanny x-men 024 
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : UseFuzzy is  : None
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : ComVersChk : 3
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 1. Bracket Word: 2014
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : year detected: 2014
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : year looking for: 2014
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 2014 - right years match baby!
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : 2. Bracket Word:   digital zone-empire
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : Scanner detected:   digital zone-empire
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : original nzb comic and issue: uncanny x-men 024 
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : [X-Men] I have found a -  within the nzbname @ position: 9
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : There is a - present in the series title. Ignoring position: 9
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : adjusted nzb comic and issue: uncanny x men 024 
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : adjusting from: uncanny x men 024  to: 024
04-Apr-2015 18:05:22 - DEBUG   :: Thread-16 : ['uncanny', 'x', 'men', '024'] nzb series word count: 3
